src/extension.ts

```diff
@@ -40,7 +40,7 @@ export function activate(context: vscode.ExtensionContext) {
 
   Logger.info(tag, 'one-vscode activate OK');
 
-  new OneExplorer(context);
```

Comment:
:bell: Details

Created class object is no where used

Reply:
Do you provide any the `Details`? Or is it just to emphasize..?

Reply:
Actually, I took it after the other examples. So I am worried if exporting `OneExplorer` actually matters. For example, unit testing.

Reply:
> Do you provide any the Details? Or is it just to emphasize..?

It was because of https://github.com/Samsung/ONE-vscode/pull/931#issuecomment-1173475835...!

> Actually, I took it after the other examples. So I am worried if exporting OneExplorer actually matters. For example, unit testing.

This PR aims to remove unused object, not about exporting.
And surprisingly, `OneExplorer` is only used in `extension.ts` :)

Reply:
Sure. :-D It needs rebase now..